transactions/set-permissionless/jul-5/README.md

```diff
@@ -0,0 +1,63 @@
+# Enable permissionless contract deployment
+> July 5th, 2022
+
+## Dapper Labs Builds
+
+```sh
+flow transactions build ./templates/set_permissionless.cdc \
```

Comment:
I think the file name is a bit confusing. Set permissionless, and then we're setting the bool to false. This is because the actual bool in the path is "isContractDeploymentRestricted", maybe we can set the cadence file name to be the same?

Reply:
Updating and will push